code/iaas/model/src/main/java/io/cattle/platform/core/model/Service.java

```diff
@@ -146,6 +146,17 @@
 	@javax.persistence.Column(name = "environment_id", precision = 19)
 	public java.lang.Long getEnvironmentId();
 
+	/**
+	 * Setter for <code>cattle.service.vip</code>.
+	 */
+	public void setVip(java.lang.String value);
+
+	/**
+	 * Getter for <code>cattle.service.vip</code>.
+	 */
+	@javax.persistence.Column(name = "vip", length = 255)
+	public java.lang.String getVip();
```

Comment:
@ibuildthecloud primary to me implies that there would be secondary...Also if we change it, then VIP service relationship to this ip address would be unclear.